conf/modules/fdm_gazebo.xml

```diff
@@ -83,8 +83,8 @@
   <header/>
   <makefile target="nps">
   	<raw>
-  	nps.CXXFLAGS  += $(shell pkg-config gazebo --cflags)
-    nps.LDFLAGS += $(shell pkg-config gazebo --libs)
+  	  nps.CXXFLAGS += $(shell pkg-config gazebo --cflags)
```

Comment:
you can use the `flag` node, see https://github.com/paparazzi/paparazzi/blob/master/conf/modules/cv_opencvdemo.xml

Reply:
How do I get that to work with the shell command? I will make the update in #2120.

Reply:
hum, right `flag` adds a `-`. I guess I'll add a `shell` node or attribute when I found some time.